esmvaltool/utils/recipe_test_workflow/recipe_test_workflow/app/install_cold/bin/install_cold.sh

```diff
@@ -31,3 +31,7 @@ fi
 # (which is done via done via 'stderr').
 git clone -q -b "${BRANCH}" "${ESMVALTOOL_URL}" "${ESMVALTOOL_DIR}"
 git clone -q -b "${BRANCH}" "${ESMVALCORE_URL}" "${ESMVALCORE_DIR}"
+
+# Manually and temporarily install imagehash (it will be included in the
+# next ESMValTool community environment).
+rtw-env pip install imagehash -t "${CYLC_WORKFLOW_RUN_DIR}/lib/ESMValTool"
```

Comment:
`imagehash` should not be installed in the same directory as ESMValTool. Use the following (and see [this comment](https://github.com/ESMValGroup/ESMValTool/pull/2803#pullrequestreview-1127646732) for more information):
```suggestion
rtw-env pip install imagehash -t "${CYLC_WORKFLOW_RUN_DIR}/lib/python"
```